queue_services/entity-filer/src/entity_filer/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.72.0'  # pylint: disable=invalid-name
+__version__ = '2.73.0'  # pylint: disable=invalid-name
```

Comment:
I don't think i see any filer changes since the last deploy

Reply:
Yes. You are right.  I will revert that change